src/contracts/BMath.sol

```diff
@@ -7,15 +7,19 @@ import {BNum} from './BNum.sol';
 
 contract BMath is BBronze, BConst, BNum {
```

Comment:
Given BMath is defining public methods, what's the rationale for not having an `IBMath` with all the definitions and natspec? are these methods ever called externally?

If that's not the case, perhaps in the future we can save deployment gas by making them internal

Reply:
I agree. Didn't add the interface because I assume that this is going to be a library/abstract. cc @wei3erHase

Reply:
tried making it into an a library and then we start having issues with `stack-too-deep` errors, i guess we could add interfaces to this contract, and then make `IBPool is IBMath`